sources/deny.toml

```diff
@@ -25,6 +25,7 @@ allow = [
 
 exceptions = [
     { name = "unicode-ident", version = "1.0.4", allow = ["MIT", "Apache-2.0", "Unicode-DFS-2016"] },
+    { name = "webpki-roots", allow = ["MPL-2.0"], version = "*" },
```

Comment:
We shouldn't allow `webpki-roots` to come in, as it means that something might not be using the CA certificates we manage.

Reply:
Whoops. I meant to remove this in my last push, but missed it. It was being pulled in to support the `rustls-webpki` feature, but with that feature removed it won't get pulled in.